web-components/src/components/combobox/ComboBox.ts

```diff
@@ -1007,6 +1007,12 @@ export namespace ComboBox {
             });
           }
           break;
+          case Key.ArrowLeft:
+            case Key.ArrowRight:
```

Comment:
Please correct the indentation of this line.

Reply:
i have updated the git